pom.xml

```diff
@@ -24,23 +24,57 @@
 
   <properties>
     <java.version>1.8</java.version>
+    <slf4j.version>1.6.6</slf4j.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-batch</artifactId>
+        <exclusions>
+          <exclusion>
+            <groupId>org.slf4j</groupId>
```

Comment:
We should discuss logging framework options .. and pick one and apply it across all code bases so we don't have to address these conflicts. (not a change request, just thinking out loud)